db/migrate/20210812004325_add_voting_token_to_members.rb

```diff
@@ -0,0 +1,11 @@
+class AddVotingTokenToMembers < ActiveRecord::Migration[5.2]
+  def up
+    add_column :members, :voting_token, :string, unique: true
+    Member.find_each { |member| member.regenerate_voting_token }
```

Comment:
The app is a tiny little one so it won't be a problem in the future (we can just remove it after the AGM), but for the record I prefer something like:

```ruby
execute %[
  update members
  set voting_token = md5(random()::text || clock_timestamp()::text)
  where voting_token is null;
]
```

:v: